src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -621,14 +621,12 @@ private List<Integer> getEmptyLines(DetailAST ast) {
      */
     private static List<Integer> getEmptyLinesToLog(List<Integer> emptyLines) {
         final List<Integer> emptyLinesToLog = new ArrayList<>();
-        if (emptyLines.size() >= 2) {
-            int previousEmptyLineNo = emptyLines.get(0);
```

Comment:
This follows the same logic as the previous one.

The for loop still investigates the 1st item and it is impossible to match it as I explained before to add to the result.